### DISCUSSION TITLE
Does ory encrypt identity date at rest

### DISCUSSION THREAD

**Discussion by YourTechBud** [ory/kratos#3025] [Q&A] [answered]
Does ory kratos provide the ability to encrypt certain pii in the identity data?

**Comment by vinckr** [comment, marked as answer]
Can you elaborate a bit more on the use case? 

You have the option to use admin [metadata](https://www.ory.sh/docs/kratos/manage-identities/managing-users-identities-metadata) which is not visible to the enduser.

Ory Kratos does not encrypt data at rest out-of-the-box, [Ory Network](https://www.ory.sh/security/) does this.

**Reply by YourTechBud** [reply]
Our identity data contains Personally Identifiable infor about our users... Things like email, name, address. As per compliance policies, such data needs to be encrypted at rest.

**Reply by hyperknot** [reply]
If you use a DB which is encrypted at rest you are compliant I believe. Pretty much all hosted DBs are encrypted at rest.

**Reply by vinckr** [reply]
Could not have said it better, Ory Network uses cockroachdb in the backend to provide this. When you are hosting Ory Services choose a DB provider or solution that provides encryption.
